.github/workflows/README.md

```diff
@@ -0,0 +1,155 @@
+# AlphaX Release Process
+These [reusable workflow](https://docs.github.com/en/actions/sharing-automations/reusing-workflows) aim at unifying the release process for all AlphaX packages.
+It automates the creation of a draft release on GitHub, the building of installers for different platforms,
+testing of the installers, the publishing of the package on PyPi and (optionally) the publishing on Docker Hub.
+
+The first part of this Readme explains the release process itself.
+The second part shows how to incorporate this pipeline into your repository.
+The last part explains how to further develop this pipeline.
+
+## Release a new version
+
+### Release a new version
+Note: these instructions assume that the release is done from the `main` branch.
+
+1. Bump the version locally to (e.g. to `X.Y.Z`), e.g. using `bumpversion`, and push this change to `main`.
+2. Manually run the 'Create Draft Release' workflow 
+(the one referencing [this workflow](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml))
+in your repository to create a new draft release on GitHub 
+(in GitHub UI: "Actions" -> Workflow "Create Draft Release" -> "Run Workflow").
+
+When running the workflow you will be prompted to specify
+- the full commit hash to release (e.g. the latest commit to `main`), and
+- the release tag (e.g. `vX.Y.Z`, note the prefixed `v`). The version in this tag must match the code version.
```

Comment:
just out of curiosity, why is the manual specification necessary?

Reply:
I found it a bit too dangerous to directly get this from the code, but we could also do that .. let's address this in the next Monday meeting